app/views/claims/pages/start.html.erb

```diff
@@ -0,0 +1,48 @@
+<div class="govuk-width-container">
```

Comment:
Missing `<%= content_for :page_title, ... %>`

Reply:
🤦🏻

Reply:
Oh, I didn't put it in as the default one is correct `Claim funding for mentors - GOV.UK`
![Screenshot from 2024-03-08 14-52-11](https://github.com/DFE-Digital/itt-mentor-services/assets/11318084/554b8e4f-97ff-4735-b872-978fdd9cc98f)